### PR TITLE
Aug 25 editorial edition

### PR DESCRIPTION
**Pull request by pedro-hos**

### CODE REVIEW

File: src/content/posts/weekly-2022-08-25.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+link:https://quarkus.io/blog/l10n-of-quarkusio/[Hola, 你好, こんにちは Quarkus], by Max Rydahl Andersen
+
+Since the launch of Japanese link:http://ja.quarkus.io/[ja.quarkus.io], we got contributors translating the website into (simplified) Chinese at link:http://cn.quarkus.io[cn.quarkus.io] and recently we added Spanish at link:http://es.quarkus.io[es.quarkus.io] too. These sites are now accessible via the drop-down "globe" menu in the top-right corner.
+
+== Using RHOSAK from WildFly
+
+link:https://www.wildfly.org/news/2022/08/19/RHOSAK/[Using RHOSAK from WildFly], by Kabir Khan
+
+In this post Kabir will guide how to write a simple application which sends and receives messages to/from a Kafka instance using **RHOSAK**. RHOSAK - Red Hat OpenShift Streams for Apache Kafka, is a cloud service hosted by Red Hat which makes setting up, managing and scaling Apache Kafka instances very easy
```
Comment (oraNod) [inline src/content/posts/weekly-2022-08-25.adoc:26]:
```suggestion
In this post Kabir shows how to write a simple application which sends and receives messages to/from a Kafka instance using **RHOSAK**. RHOSAK - Red Hat OpenShift Streams for Apache Kafka, is a cloud service hosted by Red Hat which makes setting up, managing, and scaling Apache Kafka instances very easy 
```

File: src/content/posts/weekly-2022-08-25.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+link:https://www.wildfly.org/news/2022/08/19/RHOSAK/[Using RHOSAK from WildFly], by Kabir Khan
+
+In this post Kabir will guide how to write a simple application which sends and receives messages to/from a Kafka instance using **RHOSAK**. RHOSAK - Red Hat OpenShift Streams for Apache Kafka, is a cloud service hosted by Red Hat which makes setting up, managing and scaling Apache Kafka instances very easy 
+
+== Getting started with AtlasMap
+
+link:http://www.mastertheboss.com/java/getting-started-with-atlasmap/[Getting started with AtlasMap], by Francesco Marchioni
+
+In this article, Francesco shows the basics of AtlasMap data mapping solution. He also have covered the basic set up of the Web UI and how to use it to create a minimal mapping file. Finally, he have discussed two simple ways to use AtlasMap in your Java projects.
```
Comment (oraNod) [inline src/content/posts/weekly-2022-08-25.adoc:32]:
```suggestion
In this article, Francesco shows the basics of the AtlasMap data mapping solution. He also covers the basic set up of the Web UI and how to use it to create a minimal mapping file. Finally, he discusses two simple ways to use AtlasMap in your Java projects.
```

File: src/content/posts/weekly-2022-08-25.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+link:https://developers.redhat.com/articles/2022/08/25/optimize-loops-long-variables-java#why_optimization_was_added_for_long_variables[Optimize loops with long variables in Java], by Roland Westrelin
+
+The just-in-time (JIT) compiler in OpenJDK improves Java performance through a number of optimizations, particularly in loops. Until recently, many optimizations worked only when the loop index was an int variable. In this article, Roland shows how the HotSpot virtual machine was upgraded to add the same optimizations for long variables.
+
+== A DMN FEEL HANDBOOK
+
+link:https://blog.kie.org/2022/08/a-dmn-feel-handbook.html[A DMN FEEL handbook], by Matteo Mortari
+
+In this announcement, Matteo are introducing an (experimental) DMN FEEL handbook, an helpful companion for your DMN modeling activities! You can access the FEEL handbook at link:https://kiegroup.github.io/dmn-feel-handbook/#dmn-feel-handbook[DMN FEEL handbook]
```
Comment (oraNod) [inline src/content/posts/weekly-2022-08-25.adoc:44]:
```suggestion
In this announcement, Matteo introduces an (experimental) DMN FEEL handbook, a helpful companion for your DMN modeling activities! You can access the FEEL handbook at link:https://kiegroup.github.io/dmn-feel-handbook/#dmn-feel-handbook[DMN FEEL handbook]
```

**Review by oraNod — COMMENTED** [review]
@pedro-hos Looks good! Some minor nits.

**Comment by pedro-hos** [comment]
Hey, @oraNod Thanks for your review! I've just pushed the fixes!

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks @pedro-hos

**Pull request merged**